speech/cloud-client/src/main/java/com/example/speech/QuickstartSample.java

```diff
@@ -50,6 +50,7 @@ public static void main(String... args) throws Exception {
         .setEncoding(AudioEncoding.LINEAR16)
         .setSampleRateHertz(16000)
         .setLanguageCode("en-US")
+        .setEnableWordTimeOffsets(false)
```

Comment:
I'm pretty sure we don't want to set an optional parameter to its default value in the quickstart example. Thoughts?